### DISCUSSION TITLE
Training Stress Score

### DISCUSSION THREAD

**Discussion by hambo1** [cagnulein/qdomyos-zwift#2859] [Ideas]
Could we please have a calculation for training Stress score (TSS) and a tile to display tss?

**Comment by cagnulein** [comment]
Sure there is already an open ticket here
https://github.com/cagnulein/qdomyos-zwift/issues/516


Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno lun 2 dic 2024 alle 20:38 hambo1 ***@***.***> ha
scritto:

**Comment by cagnulein** [comment]
@hambo1 do you like a tile or in the summary email of qz?

**Reply by cagnulein** [reply]
@hambo1 i did a first implementation. are you on ios or android?

**Comment by hambo1** [comment]
A tile please so it’s visible during a ride.

Thanks Matt

**Reply by cagnulein** [reply]
ok done

**Comment by hambo1** [comment]
I'm on Android thanks

**Reply by cagnulein** [reply]
on it's building https://github.com/cagnulein/qdomyos-zwift/actions/runs/12161134329

Google is very slow in these days, reviewing the app on the store, even the beta ones. For this reason, in order to check right now if the patch that I did is right, you can use the android build in the link (check at the bottom of the page in about 1 hour). In order to test it you need first to remove the current QZ version on your phone, install the one that you downloaded.
This version is a 15 minutes trial (you can restart it as many times as you want) and I can unlock it if you want, send me an email to roberto.viola83@gmail.com mentioning this ticket and that you are on android.
Let me know if you have any questions

**Comment by hambo1** [comment]
Brilliant thanks I can test tomorrow

**Comment by hambo1** [comment]
Looks like the build is showing status failure?

**Comment by cagnulein** [comment]
nope download the apk from here
https://github.com/cagnulein/qdomyos-zwift/actions/runs/12161134329/artifacts/2273737303

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno mer 4 dic 2024 alle ore 16:36 hambo1 ***@***.***>
ha scritto:

> Looks like the build is showing status failure?
>

**Comment by cagnulein** [comment]
@hambo1 Did you check my pr?

**Comment by hambo1** [comment]
Hi Roberto,

Yes sorry I replied to your email on the 4th December tested and everything
worked ok thank you.

Regards

Matt

Hi Roberto,

Installed the test version and added the tss tile successfully. Also
checked my ftp setting. Had a very short ride on Mywhoosh Parramatta and
could see the TSS value increasing during the ride. Other controls virtual
shifting worked perfectly.

Thanks

Matt


On Wed, 4 Dec 2024, 15:39 Roberto Viola, ***@***.***> wrote:
nope download the apk from here
https://github.com/cagnulein/qdomyos-zwift/actions/runs/12161134329/artifacts/2273737303

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud
